spark-commons/src/main/scala/za/co/absa/spark/commons/sql/functions.scala

```diff
@@ -63,4 +64,9 @@ object functions {
     lit(None.orNull)
   }
 
+  def null_col(dataType: DataType):Column = {
+    lit(None.orNull).cast(dataType)
+  }
+
+
 }
```

Comment:
Please add a new line at the end and remove unnecessary new line (either line 70 or 71)